src/less/variables.less

```diff
@@ -175,10 +181,12 @@
 
 // Buttons
 //-------------------------
-@btn-default-color:           #fff;
-@btn-default-bg:              @gray-400;
+@btn-default-color:           @black;
+@btn-default-bg:              @body-bg;
+@btn-default-border:          @gray-150;
+
 
-@btn-primary-color:           @btn-default-color;
+@btn-primary-color:           @body-bg;
 @btn-primary-bg:              @brand-success;
 
 @btn-danger-color:            @btn-default-color;
```

Comment:
This needs to be white. Danger buttons are dark red and require white text, not black text.